packages/dialtone-vue2/recipes/conversation_view/time_pill/time_pill.vue

```diff
@@ -9,14 +9,13 @@
 </template>
 
 <script>
-import {} from './time_pill_constants';
-
 export default {
   name: 'DtRecipeTimePill',
 
   props: {
     /**
      * Date time display value
+     * TODO: Should this text be localized?
```

Comment:
No, this is passed in from the product, so they will translate themselves if needed.

Reply:
That's true for almost every other prop that I just replaced, I just didn't quickly find a way to do it, that's why I asked, we also have tickets related to this, so not doing it on this PR, but definitely will be localizing this in the future:

https://dialpad.atlassian.net/browse/DLT-1045
https://dialpad.atlassian.net/browse/DLT-1363
https://dialpad.atlassian.net/browse/DLT-1291

Reply:
Yeah in future we might want to actually have logic to manipulate the date, but right now the intention of this component is to just display "whatever is passed into it"